readme.md

```diff
@@ -56,7 +56,7 @@ Issues
 ------
 
 We have a few ambiguous labels on the issues page, so here are their definitions:
-* `platform: all` - An issue which applies to all supported platforms (`Bukkit`, `Sponge`, `BungeeCord`, `Forge`, and `LiteLoader`)
+* `platform: all` - An issue which applies to all supported platforms (`Bukkit`, `Sponge`, `BungeeCord`, `Forge`, `LiteLoader` and `Canary`)
```

Comment:
Oxford comma, please.
![x](http://cdn.thewritepractice.com/wp-content/uploads/2011/08/the-oxford-comma.jpg)